<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Pic">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>322</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="open">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeCam">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_Threshold">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>45</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Threshold">
       <property name="text">
        <string>Threshold Value：45</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_Contour">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Contour">
       <property name="text">
        <string>Contour Filter Value：80</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="MPP">
       <property name="placeholderText">
        <string>0.01</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Time_PerFrame_2">
       <property name="text">
        <string>Meter Per Pixel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Camera_Situation">
       <property name="text">
        <string>Camera Situation：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Position_X">
       <property name="text">
        <string>Position X-&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Position_Y">
       <property name="text">
        <string>Position Y-&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Time">
       <property name="text">
        <string>Time：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Time_PerFrame">
       <property name="text">
        <string>Time Per Frame：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Velocity">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Situation">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Pic2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>322</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pic3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>322</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
